lib/src/view/user/user_screen.dart

```diff
@@ -13,9 +13,18 @@ import 'user_activity.dart';
 import 'user_profile.dart';
 
 class UserScreen extends ConsumerWidget {
-  const UserScreen({required this.user, super.key});
+  const UserScreen._({this.lightUser, this.user, super.key});
```

Comment:
since one of them is required (but not both) we should have an `assert` to enforce it.